index.js

```diff
@@ -37,6 +52,7 @@ module.exports = function serialize(obj, options) {
         options = {space: options};
     }
 
+    var uid = options.uid || UID;
```

Comment:
I need a deterministic value for `UID` for the regression test I added, the other option is to monkeypatch `crypto.randomBytes()` in the tests like I was doing with `Math.random()`, but that felt a little brittle. I'm fine with whichever you prefer